container.php

```diff
@@ -196,7 +196,18 @@ public function onRoute(ActionEvent $actionEvent)
             $buildings = $container->get(BuildingRepositoryInterface::class);
 
             return function (Command\RegisterNewBuilding $command) use ($buildings) {
-                $buildings->add(Building::new($command->name()));
+                $buildings->store(Building::new($command->name()));
+            };
+        },
+        Command\CheckInUser::class => function (ContainerInterface $container) : callable {
+            $buildings = $container->get(BuildingRepositoryInterface::class);
+
+            return function (Command\CheckInUser $command) use ($buildings) {
+                $building = $buildings->get($command->buildingId());
+
+                $building->checkInUser($command->username());
+
+                $buildings->store($building);
```

Comment:
While calling `$buildings->store()` is not necessary, since Prooph has its own concept of identity map and transaction boundaries, this makes it clear that we are saving `$building` after having modified it